benchmark/src/bin/ruben.rs

```diff
@@ -5,81 +5,72 @@ use admission_control_proto::proto::{
     admission_control::SubmitTransactionRequest, admission_control_grpc::AdmissionControlClient,
 };
 use benchmark::{
-    ruben_opt::{Executable, Opt},
+    ruben_opt::{Opt, TransactionPattern},
     txn_generator::{
-        gen_accounts, gen_pairwise_transfer_txn_requests, gen_ring_transfer_txn_requests,
+        LibraRequest, LibraRequestGenerator, PairwiseTransferTxnGenerator, RingTransferTxnGenerator,
     },
     Benchmarker,
 };
 use client::AccountData;
 use grpcio::{ChannelBuilder, EnvBuilder};
-use libra_wallet::wallet_library::WalletLibrary;
 use logger::{self, prelude::*};
 use metrics::metric_server::start_server;
 use std::sync::Arc;
 
+/// Benchmarker is not ready to take LibraRequest(both write and read requests) yet.
+/// This simple conversion is a temporary fix.
+fn convert_libra_requests_to_submit_txn_requests(
```

Comment:
could you add a TODO here to remove this temporary function later?

Reply:
Done.